packages/bpk-component-infinite-scroll/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bpk-component-infinite-scroll",
+  "version": "0.0.0",
+  "description": "Backpack infinite scroll component.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^15.0.0 || ^16.0.0"
+  },
+  "dependencies": {
+    "bpk-mixins": "^17.9.0",
+    "intersection-observer": "^0.5.0",
+    "prop-types": "^15.5.8"
```

Comment:
you need to add `"bpk-react-utils": "^2.6.1"` since you're now using it in your component